metrics/next-linux-gcc8-release-metrics.json

```diff
@@ -0,0 +1,18 @@
+{
+    "asset_path": "../mapbox-gl-js/test/integration",
+    "base_test_path": "tests",
+    "expectation_paths": [
+        "expectations/platform-all"
+    ],
+    "ignore_paths": [
+        "ignores/platform-all.json",
+        "ignores/platform-linux.json"
+    ],
+    "metric_path": "next-linux-gcc8-release",
+    "probes": [
+        "probeGFX",
+        "probeMemory",
```

Comment:
Memory probe for small render tests, would be flaky and trigger metric regeneration all the time, same on android bot. That's why we had [real-world](https://github.com/mapbox/mapbox-gl-native/pull/16052/commits/e91a87d777e0f8ffddc1cd43b7777dc50f23f909#diff-65b1263b1c34cc46f4ed7029c15c3425L10) filter.